app/services/explorer.js

```diff
@@ -760,12 +757,14 @@ export default Ember.Service.extend({
      * @param {DS.Store} store
      * @return {Ember.RSVP.Promise<BucketType>}
      */
-    getBucketTypeWithBucketList(bucketType, cluster, store) {
-        return this.getBucketList(cluster, bucketType, store)
-            .then(function(bucketList) {
-                bucketType.set('bucketList', bucketList);
-                return bucketType;
-            });
+    // TODO: What is the difference between this and getBucketList???
+    getBucketTypeWithBucketList(bucketType, cluster, store, start, row) {
+        return this
+          .getBucketList(cluster, bucketType, store, start, row)
+          .then(function(bucketList) {
+              bucketType.set('bucketList', bucketList);
+              return bucketType;
+          });
```

Comment:
I'm not seeing the difference between this and `getBucketList`. Would you mind explaining how they are different.